### PR TITLE
Avoid Firefox issue 548397

### PR DESCRIPTION
**Pull request by dmlap**
[getComputedStyle returns null](https://bugzilla.mozilla.org/show_bug.cgi?id=548397) when in an iframe with display: none so be careful dereferencing it.

### CODE REVIEW

**Comment by incompl** [comment]
LGTM

File: test/videojs-hls_test.js [hunk 1]
```diff
@@ -3106,6 +3106,26 @@ test('does not process update end until buffered value has been set', function()
...
+  player.src({
+    src: 'master.m3u8',
+    type: 'application/vnd.apple.mpegurl'
+  });
+  openMediaSource(player);
+  standardXHRResponse(requests.shift()); // master
+  standardXHRResponse(requests.shift()); // media
+
+  player.tech_.hls.selectPlaylist();
+  ok(true, 'should not throw');
```
Comment (gesinger) [inline test/videojs-hls_test.js]: Is this preferred over leaving a comment above selectPlaylist that an error should not be thrown?

Comment (gkatsev) [inline test/videojs-hls_test.js]: qunit requires an assertion.

Comment (gesinger) [inline test/videojs-hls_test.js]: Ah, gotcha. Thanks.

**Comment by dmlap** [comment]
Weird Firefox test failure in Travis. Investigating.

**Pull request merged**